Tutorials/a1_perception_setup.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: default
+title: Set Up Perception Pipeline for A1
+parent: Tutorials
+nav_order: 99 
+mathjax: true
+---
+**Last Updated:** 2021-07-20
+
+# Setting Up Perception Pipeline for A1
+This tutorial goes through steps to set up the perception pipeline for the A1 quadruped robot, which includes installing drivers for the RealSense D435i camera, installing `pytorch` and `torchvision` specifically for the TX2 platform.
+
+## Connecting to the Robot
+By default, the A1 robot has a built-in wifi HotSpot named `UnitreeRoboticsA1-XXX`, where `XXX` is the robot serial number. To access the onboard TX2, connect to the wifi hotspot (default password is `00000000`). Then you can `ssh` into the robot by running:
+
+`ssh unitree@192.168.123.12`
+
+with password `123`.
```

Comment:
I would caution against putting 'non-default' password here! Perhaps set a 'standard' password that is known internally, but not shared on the internet.

Reply:
These are default password from Unitree, so there isn't much issue in sharing it.